administration/src/bp-modules/applications/ApplicationsOverview.tsx

```diff
@@ -178,13 +184,38 @@ export class ApplicationViewComponent extends React.Component<{
   }
 }
 
+const sortByStatus = (a: number, b: number): number => a - b
+const sortByDateAsc = (a: Date, b: Date): number => a.getTime() - b.getTime()
+
+const getApplicationStatus = (status: number[]): ApplicationStatus => {
+  if (status.every(val => val === VerificationStatus.Verified)) return ApplicationStatus.fullyVerified
+  if (status.every(val => val === VerificationStatus.Rejected)) return ApplicationStatus.fullyRejected
+  return ApplicationStatus.ambiguous
+}
+
 const ApplicationsOverview = (props: { applications: Application[] }) => {
   const [updatedApplications, setUpdatedApplications] = useState(props.applications)
   const { applicationIdForPrint, printApplicationById } = usePrintApplication()
 
+  // Applications will be sorted by unique status which means fully verified/rejected and within this status by creation date asc
+  const sortApplications = useMemo(
+    () =>
+      (applications: Application[]): Application[] =>
+        applications
+          .map(application => ({
+            ...application,
+            status: getApplicationStatus(application.verifications.map(getStatus)),
+          }))
+          .sort(
+            (a, b) =>
+              sortByStatus(a.status, b.status) || sortByDateAsc(new Date(a.createdDate), new Date(b.createdDate))
+          ),
+    [props.applications]
+  )
```

Comment:
This now memoizes the function which sorts the applications.
This is not what we want; we want to memoize the actual sorted array.

Reply:
Ah okay I got it. Gonna change that